source/common/stats/stat_merger.h

```diff
@@ -41,20 +41,43 @@ class StatMerger {
   };
 
   StatMerger(Stats::Store& target_store);
+  ~StatMerger();
 
-  // Merge the values of stats_proto into stats_store. Counters are always straightforward
-  // addition, while gauges default to addition but have exceptions.
+  /**
+   * Merge the values of stats_proto into stats_store. Counters are always
+   * straightforward addition, while gauges default to addition but have
+   * exceptions.
+   *
+   * @param counter_deltas map of counter changes from parent
+   * @param counter_deltas map of gauge changes from parent
```

Comment:
typo. I think this is gauge values also?

Reply:
Yes that was a typo; fixing.